sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValue.cs

```diff
@@ -16,17 +16,140 @@ namespace Azure.AI.FormRecognizer.Models
         private readonly IReadOnlyList<ReadResult> _readResults;
 
         internal FieldValue(FieldValue_internal fieldValue, IReadOnlyList<ReadResult> readResults)
+            : this()
         {
             ValueType = fieldValue.Type;
             _fieldValue = fieldValue;
             _readResults = readResults;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FieldValue"/> structure.
+        /// </summary>
+        /// <param name="value">The actual field value.</param>
+        /// <param name="isPhoneNumber">Whether or not this value represents a phone number.</param>
+        internal FieldValue(string value, bool isPhoneNumber = false)
```

Comment:
should we change the `bool` for `FieldValueType` instead, so that if the service adds more "types" represented by a string, the user can keep using this method?

Reply:
The method is internal, so users won't use it directly anyway (they'll call `Factory.FieldValueWithStringType` and `Factory.FieldValueWithPhoneNumberType`). I think your comment is reasonable, but I prefer having to worry about that only if the service makes such addition. I'm not a fan of the idea of having invalid method calls, such as:
```cs
_ = new FieldValue("something", FieldValueType.List);
```
But it doesn't me worry that much if it's internal.